build-support/functions/20-build.sh

```diff
@@ -139,59 +139,14 @@ function build_ui {
    # Copy UI over ready to be packaged into the binary
    if test ${ret} -eq 0
    then
-      rm -rf ${1}/pkg/web_ui
-      mkdir -p ${1}/pkg
-      cp -r ${1}/ui/dist ${1}/pkg/web_ui
+      rm -rf ${1}/agent/uiserver/dist
```

Comment:
Not 100% sure whats happening here, but I get a stray:

<img width="254" alt="Screenshot 2021-09-27 at 11 09 37" src="https://user-images.githubusercontent.com/554604/134890626-ec5a824c-00b3-415f-b260-1e9e2790f698.png">

After running `make ui` 🤔 not sure how thats happened? Oh wait I guess that REMOVEME.txt must have been commited somewhere?

Reply:
Yep! The `dist/` directory needs to exist for this PR to compile but I didn't check in the artifacts so it's a placeholder just so that git doesn't ignore `dist/`